recipes/gitdb2/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.0.2" %}
+{% set sha256 = "f2e36d7561e91f30a6a44858756dc020d8f1e81ca6e4185979d5c6c24c648070" %}
+
+package:
+  name: gitdb
```

Comment:
Should this have been called `gitdb2`? Given it overlapped with an existing feedstock, `staged-recipes` didn't overwrite the feedstock. Maybe it should go in as a branch if `gitdb` is the name needed. Alternatively this recipe may need to be readded with a name change. Thoughts @ngoldbaum?